<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VWindowMaterials</class>
 <widget class="QMainWindow" name="VWindowMaterials">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База материалов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="newMaterialButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Новый материал</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Импорт базы материалов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Экспорт базы материалов</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="materialsLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Материалы</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QFrame" name="parametersFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="param1Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Параметр 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="param1Edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="param2Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Параметр 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="param2Edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="param3Label">
          <property name="text">
           <string>Параметр 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="param3Edit">
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="param4Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Параметр 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="param4Edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="saveRefreshHorizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="text">
             <string>Отменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Удалить материал</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="materialsListWidget"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>newMaterialButton</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>param1Edit</tabstop>
  <tabstop>param2Edit</tabstop>
  <tabstop>param3Edit</tabstop>
  <tabstop>param4Edit</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>materialsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param1Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param2Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param4Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param3Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
